corehq/apps/cleanup/tasks.py

```diff
@@ -16,34 +16,33 @@
 from corehq.apps.cleanup.utils import get_cutoff_date_for_data_deletion
 from corehq.apps.domain.models import Domain
 from corehq.apps.hqwebapp.tasks import mail_admins_async
-from corehq.form_processor.models import XFormInstance
+from corehq.form_processor.models import CommCareCase, XFormInstance
 
 UNDEFINED_XMLNS_LOG_DIR = settings.LOG_HOME
 
 logger = logging.getLogger(__name__)
 
 
-# @periodic_task(run_every=crontab(minute=0, hour=0), queue=getattr(settings, 'CELERY_PERIODIC_QUEUE', 'celery'))
-def permanently_delete_eligible_data(dry_run=False):
+@periodic_task(run_every=crontab(minute=0, hour=0), queue=getattr(settings, 'CELERY_PERIODIC_QUEUE', 'celery'))
+def permanently_delete_and_tombstone_eligible_data(dry_run=False):
```

Comment:
I have mixed feelings about the name change. Not that I loved the previous name, I just don't think we need to mention the tombstone part in the method name. Especially if this grows into the task that is used to kick off _all_ periodic deletions, in which case it isn't guaranteed to tombstone every object it deletes. It seems like we are heading in a direction where some things are tombstoned, and some things aren't, and it is up to the specific models to decide if they want to tombstone on deletion. Tombstoning has no impact on this method otherwise.